demo/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,7 @@
     <string name="edt_lastname">Last Name</string>
     <string name="edt_year">Year</string>
     <string name="edt_height">Height (m)</string>
+    <string name="edt_birth">Birth date E.g 18/12/1995</string>
```

Comment:
Is this your birthday?!  Same date as mine! :)

Reply:
haha, no it's just a random number I wrote but interesting.

Reply:
What a coincidence ;)